src/test/java/uk/gov/hmcts/ccd/endpoint/std/CaseAccessEndpointIT.java

```diff
@@ -44,12 +49,15 @@
     private MockMvc mockMvc;
     @SpyBean
     private AuditRepository auditRepository;
+    @MockBean
+    private RoleAssignmentCategoryService roleAssignmentCategoryService;
```

Comment:
```suggestion
```
Should use WireMock for IT instead to test the e2e flow (bearing in mind it'll now need to auth with the new Data Store system user)